cluster/addons/fluentd-elasticsearch/fluentd-es-configmap.yaml

```diff
@@ -115,7 +115,6 @@ data:
       @type tail
       path /var/log/containers/*.log
       pos_file /var/log/es-containers.log.pos
-      time_format %Y-%m-%dT%H:%M:%S.%NZ
```

Comment:
instead of removing this line, can you add `time_type string`?  this will cause the specified `time_format` to be applied

Reply:
@coffeepac 
Thanks. Looks like `time` parameters (`time_type`, `time_format`,..) are only applicable to `<parse>` section of source plugin `tail`. If we use this params outside `parse` section in `tail` plugin then fluentd seems ignoring these params and we are getting the warnings.

I got the below errors/warnings when I tried to add `time_type string` , please correct me if I am wrong.

```
2018-08-28 17:28:59 +0000 [warn]: parameter 'time_format' in <source>
  @id fluentd-containers.log
  @type tail
  path "/var/log/containers/*.log"
  pos_file "/var/log/es-containers.log.pos"
  time_format %Y-%m-%dT%H:%M:%S.%NZ
  time_type string
  estimate_current_event false
  refresh_interval 20
  tag "raw.kubernetes.*"
  read_from_head true
  <parse>
    @type "multi_format"
    <pattern>
      format json
      time_key "time"
      time_format "%Y-%m-%dT%H:%M:%S.%NZ"
      time_type string
    </pattern>
    <pattern>
      format /^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$/
      time_format "%Y-%m-%dT%H:%M:%S.%N%:z"
      expression ^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$
      ignorecase false
      multiline false
    </pattern>
  </parse>
</source> is not used.
2018-08-28 17:28:59 +0000 [warn]: parameter 'time_type' in <source>
  @id fluentd-containers.log
  @type tail
  path "/var/log/containers/*.log"
  pos_file "/var/log/es-containers.log.pos"
  time_format %Y-%m-%dT%H:%M:%S.%NZ
  time_type string
  estimate_current_event false
  refresh_interval 20
  tag "raw.kubernetes.*"
  read_from_head true
  <parse>
    @type "multi_format"
    <pattern>
      format json
      time_key "time"
      time_format "%Y-%m-%dT%H:%M:%S.%NZ"
      time_type string
    </pattern>
    <pattern>
      format /^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$/
      time_format "%Y-%m-%dT%H:%M:%S.%N%:z"
      expression ^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$
      ignorecase false
      multiline false
    </pattern>
  </parse>
</source> is not used.
```

Reply:
ahh, okay, I was misreading the docs then.

looks good!